test/indexTest.js

```diff
@@ -0,0 +1,11 @@
+var assert = require("assert")
+var server = require("./testServer")
```

Comment:
Hadn't really considered the value, it was just the way I'd assumed you had to test web servers. Thanks Rails. I'd also assumed looking at your code that there would be some resource conflicts if you had a dev server running and then ran the tests at the same time. Is that not the case?